Content.Server/Construction/ConstructionSystem.Machine.cs

```diff
@@ -66,8 +66,8 @@ private void CreateBoardAndStockParts(EntityUid uid, MachineComponent component)
             throw new Exception($"Entity with prototype {component.Board} doesn't have a {nameof(MachineBoardComponent)}!");
         }
 
-        // Frontier: Only bind the board if the machine itself has the BindToStationComponent and the board doesn't already have BindToStationComponent  
-        if (HasComp<BindToStationComponent>(uid) && board != null)
+        // Frontier: Only bind the board if the machine itself has the StationBoundObjectComponent and the board doesn't already have StationBoundObjectComponent
+        if (HasComp<StationBoundObjectComponent>(uid) && board != null)
```

Comment:
To be honest I'm not sure where this path comes up normally, all the original seemed to do was cause those annoying weird desyncs between internal board being bound and external board being unbound.